packages/app/src/domain/tested/reproduction-chart-tile.tsx

```diff
@@ -33,7 +33,23 @@ export function ReproductionChartTile({
     0,
     data.values.findIndex((x) => !isPresent(x.index_average))
   );
+
+  /**
+   * Calculate the highest tick and round it up on the closet number on the 0.5 scales.
+   * Then create an array with every tickStep till the highestTick
+   * to use for the tickValues in the chart.
+   */
+  const tickStep = 0.5;
+
   const last_value = last(values) as NationalReproductionValue;
+  const maxIndexAverage = Math.max(
+    ...values.map((x) => x.index_average).filter(isPresent)
+  );
+  const highestTick = Math.ceil(maxIndexAverage * 2) / 2;
```

Comment:
```suggestion
  const highestTick = Math.ceil(maxIndexAverage);
```
Just wondering, this produced different results?

Reply:
It does, without it rounds up to a whole number (3). And otherwise, it rounds to the closest 0.5 value (2.5).